openwisp_monitoring/device/templates/admin/config/device/change_form.html

```diff
@@ -312,66 +312,66 @@ <h2>{% trans 'Neighbors' %}</h2>
 </div>
 {% endif %}
 {% if not add %}
-<div class="inline-group" id="ow-graph-container">
+<div class="inline-group" id="ow-chart-container">
     <fieldset class="module">
-        <h2>{% trans 'Monitoring Graphs' %}</h2>
-        <div id="ow-graph-time">
+        <h2>{% trans 'Monitoring Charts' %}</h2>
```

Comment:
Let's shorten this to just `Charts`, I'm doing the same for the other tabs (they're too long).